<launch>
<param name="/use_sim_time" value="true" />

   <node pkg="autonomous_landing" name="dense" type="depth_gt_node" output="screen" clear_params="true" > <!--launch-prefix="gdb -ex run -deletethis-args" -->
        
    </node>

   
       
    
</launch>
